python/test_python.py

```diff
@@ -121,3 +121,21 @@ class TestWriteFile:
     def test_write_file(self, docker_container):
         docker_container.wait()
         assert str(docker_container.logs(), 'UTF-8') == "BOB BARKER" # note no new line char
+
+
+class TestWriteJsonToStdout:
+    """Test cases in which JSON is written to stdout"""
+    @pytest.mark.parametrize(
+        'docker_container',
+        ['python json_stdout/arrays_nums_objs.py a bc def ghij'],
```

Comment:
It think it would be better if the args could be set as a variable elsewhere, but not sure how to pass that variable into the fixture parameter...

Reply:
Docker fixture updated in PR #22